src/components/UserInfo/UserInfo.scss

```diff
@@ -0,0 +1,5 @@
+@import "./../../utils/variables.scss";
```

Comment:
Consider using relative path instead of absolute path for better maintainability. 🛣️